http://sourceware.org/ml/gdb-cvs/2013-04/msg00110.html

### src/gdb/ChangeLog	2013/04/11 14:13:42	1.15398
### src/gdb/ChangeLog	2013/04/11 16:53:01	1.15399
## -22,6 +22,9 @@
 	* gcore.in: ... here.  Remove gcore.sh comment.  Use GDB_TRANSFORM_NAME
 	and GCORE_TRANSFORM_NAME substitutions.
 
+	Fix parsing tabs in ${gdb_target_obs}.
+	* configure.tgt (gdb_have_gcore): Replace case with for and if.
+
 2013-04-11  Jan Kratochvil  <jan.kratochvil@redhat.com>
 
 	* remote.c (unpush_and_perror): Add output message final dot.
--- src/gdb/configure.tgt	2013/04/11 14:13:43	1.273
+++ src/gdb/configure.tgt	2013/04/11 16:53:01	1.274
@@ -707,11 +707,9 @@
 
 # Check whether this target supports gcore.
 # Such target has to call set_gdbarch_find_memory_regions.
-case " ${gdb_target_obs} " in
-  *" linux-tdep.o "*)
+gdb_have_gcore=false
+for t in x ${gdb_target_obs}; do
+  if test "$t" = linux-tdep.o; then
     gdb_have_gcore=true
-    ;;
-  *)
-    gdb_have_gcore=false
-    ;;
-esac
+  fi
+done
